symengine/dict.h

```diff
@@ -10,7 +10,7 @@
 
 #include <gmpxx.h>
 #if defined(HAVE_SYMENGINE_PIRANHA)
-#include <piranha/piranha.hpp>
+#include <piranha.hpp>
```

Comment:
Ah yes!
I didn't get what you suggested on gitter (to fix this). Could you elaborate?